IDEAS/Fluid/BaseCircuits/FluidCollector.mo

```diff
@@ -0,0 +1,50 @@
+within IDEAS.Fluid.BaseCircuits;
```

Comment:
done in https://github.com/open-ideas/IDEAS/commit/596ef15fce0ad424916b20c5734044a7014621ab